rpearce/react-medium-image-zoom
# fix: zoom inside dialog that has 'click outside' handler

## Description

Closes https://github.com/rpearce/react-medium-image-zoom/issues/746 by stopping event bubbling in situations where we click or keydown on the zoom modal. This should prevent a parent modal (or anything) with a "click outside" handler from thinking that it should close when we are merely trying to unzoom the zoomed image modal.

## Testing

1. Clone this repo
2. Checkout this branch
3. Run `npm i && npm start`
4. Go to http://localhost:6006/?path=/story/img--zoom-image-from-inside-dialog
5. Click "Open Modal"
6. Click on the image
7. Click again on the image to close it. This should _not_ close the original modal!
8. Open it again, and click on the close button in the top right. Same test as above.
9. Open it again, and use the enter or spacebar on the close button in the top right. Same test as above.
